tutorials/matching_fof.ipynb

```diff
@@ -424,6 +424,126 @@
+    "For the extragalactic catalog, ellipticity is defined as $(1-q)/(1+q)$, where $q$ is the minor-to-major axis ratio\n",
+    "(see the [SCHEMA](https://github.com/LSSTDESC/gcr-catalogs/blob/master/GCRCatalogs/SCHEMA.md#schema-for-extragalatic-catalogs)). \n",
+    "On the other hand, for the object catalog, the HSM re-Gaussianization ellipticity that we are using is defined as $(1-q^2)/(1+q^2)$\n",
+    "(see e.g., Eq. 8 of [Mandelbaum et al. 2006](https://arxiv.org/abs/astro-ph/0511164)).\n",
+    "\n",
+    "Hence their definitions are in fact different, so we need to do a conversion before we compare them.\n",
+    "With some math, we can find the conversion between the two definitions $e_{\\rm HSM~def} = \\frac{2e_{\\rm EGC~def}}{1+e_{\\rm EGC~def}^2}$."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# compare the ellipticity (smartly)\n",
+    "ellipticity_conversion = lambda e: 2*e / (1.0+e*e)\n",
+    "plt.figure(figsize=(5,5));\n",
+    "plt.scatter(ellipticity_conversion(matched_gals['ellipticity_true']), matched_gals['shape_hsm_regauss_etot'], s=1);\n",
+    "lims = [0, 1]\n",
+    "plt.plot(lims, lims, c='k', lw=0.5);\n",
+    "plt.xlabel('extragalactic ellipticity (in object def.)');\n",
+    "plt.ylabel('object ellipticity');\n",
+    "plt.xlim(lims);\n",
+    "plt.ylim(lims);"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This looks much better now! \n",
+    "\n",
+    "When you were checking the [SCHEMA](https://github.com/LSSTDESC/gcr-catalogs/blob/master/GCRCatalogs/SCHEMA.md#schema-for-extragalatic-catalogs)) file,\n",
+    "you probably have also noticed that `ellipticity_true` is the ellipticity before the shear is applied (i.e., unlensed). \n",
+    "Hence this comparison is still not an apples-to-apples comparison, as the ellipticity in the object catalog is, of course, lensed. \n",
+    "\n",
+    "According to the the [SCHEMA](https://github.com/LSSTDESC/gcr-catalogs/blob/master/GCRCatalogs/SCHEMA.md#schema-for-extragalatic-catalogs)), we should have been using `ellipticity` from the extragalactic catalog.\n",
+    "But unfortunately, this quantity is not directly available from the extragalactic catalog!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "extragalactic_cat.has_quantity('ellipticity')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Hmm, that's kind of sad, but luckily we can add our own derived quantities! "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def calc_lensed_ellipticity(e1, e2, shear1, shear2):\n",
+    "    return np.hypot(e1+shear1, e2+shear2) \n",
```

Comment:
Thanks, @rmandelb. I've updated `calc_lensed_ellipticity` in commit 918d01d to address this issue. I no longer use the small-shear approximate but use the full calculation (which I hope I implemented correctly). 

(I didn't opt to use GalSim because it doesn't do array operations.)